services/core-web/src/components/maps/MineMapLeaflet.js

```diff
@@ -1,3 +1,6 @@
+// TODO: Remove this line when file is more properly implemented.
```

Comment:
What do you mean "more properly implemented"? We shouldn't have introduced a bunch of linting issues with your changes. If there's one or two issues, disable the rule on a per line basis.

Reply:
There were a couple issues disabled per line and I figured they could be addressed in the upcoming future work. I can put them back in.

Reply:
Addressed comments in person. Thanks Ryan